src/addons/SuggestionItem.jsx

```diff
@@ -0,0 +1,71 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import Highlight from 'react-highlight-words';
+import Flex from '../styles/Flex';
+
+const highlightStyle = {
+  fontWeight: 600,
+  padding: 0,
+  backgroundColor: 'transparent',
+  color: 'inherit'
+};
+
+const SuggestionItem = ({ currentValue, suggestion }) => {
+  const { label, value, title, description, image } = suggestion;
+  if (label) {
+    // label has highest precedence
+    return typeof label === 'string' ? (
+      <div className="trim">
+        <Highlight
+          searchWords={currentValue.split(' ')}
+          textToHighlight={label}
+          autoEscape
+          highlightStyle={highlightStyle}
+        />
+      </div>
+    ) : (
+      label
+    );
+  } else if (title || image || description) {
```

Comment:
What will be the usage for this ?

To what I can see it is used when we need to pass image title and description from the parent component but the item that you are passing down will always be like:

```json
{
   "label" : "",
   "value": "",
   "source":{}
}
```

Do correct me if I am wrong.